examples/analyzers/topic_with_nullable_notifications.cs

```diff
@@ -0,0 +1,6 @@
+using LeanCode.Contracts;
+
+public class Notification1 { }
+public class Notification2 { }
+
+public class NullableNotificationTopic : ITopic, IProduceNotification<Notification1?>, IProduceNotification<Notification2?> { }
```

Comment:
How does that even compile in spite of https://github.com/leancodepl/contractsgenerator/blob/48e6ed8326684bc775ae0744cdf649d98664efb0/src/LeanCode.Contracts/IProduceNotification.cs#L5?

Reply:
If only annotations are enabled, then you only get [a warning](https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwMRQgG0wQwEaYCmABIVPkcTlFAPbA7ACWtUAzgLABQ3AbjmGLBiAXmJRCAd2IBBADwAhAPwA+ABQBKANzduAAQBMsuQBUVxSQAtCYEiZB1gGbNwDeAX31GFH7kA===).